### PR TITLE
Add Where Option For Query Getting RDS Logs

### PR DESCRIPTION
**Pull request by kikuchi-d**
I need to narrow logs on RDS.
So, I added "where" option for plugin.

### CODE REVIEW

File: lib/fluent/plugin/in_rds_log.rb [hunk 1]
```diff
@@ -82,7 +83,11 @@ def output(host)
   def query(client)
     begin
       client.query("CALL mysql.rds_rotate_#{@log_type}")
-      output_log_data = client.query("SELECT * FROM mysql.#{@log_type}_backup", :cast => false)
```
Comment (toyama0919) [inline lib/fluent/plugin/in_rds_log.rb]: Why remove 「cast => false」?

Comment (kikuchi-d) [inline lib/fluent/plugin/in_rds_log.rb]: Fixed.
I gave back ":cast => false" to client.query .

File: lib/fluent/plugin/in_rds_log.rb [hunk 2]
```diff
@@ -82,7 +83,11 @@ def output(host)
...
+      query = "SELECT * FROM mysql.#{@log_type}_backup"
```
Comment (toyama0919) [inline lib/fluent/plugin/in_rds_log.rb]: query(client) method already exists.
should be define other variable name. (ex. sql)

**Comment by kikuchi-d** [comment]
I Fixed name of local variable ("query" -> "sql")

**Pull request merged**